events/class_models/meeting_models.py

```diff
@@ -18,6 +18,9 @@ def get_queryset(self):
     def get_all_event_dates(self, event_id):
         return self.get_queryset().get_all_event_dates(event_id)
 
+    def get_meeting_dates(self, event_id):
```

Comment:
i think you should also delete it from here

Reply:
Right 😄